database/sql/V049__add_full_fee_amount.sql

```diff
@@ -0,0 +1,6 @@
+-- Limit orders need to also store the solver_fee at the time of execution.
+-- This is needed because solver_fee is not guaranteed to be constant over time for limit orders.
+
+ALTER TABLE order_execution
+ADD COLUMN solver_fee numeric(78, 0)
+;
```

Comment:
Can I add this at the end of V48 file? Not sure if it is possible since the migration is already applied in staging for V48.

Reply:
No, adding this to an already applied migration script wouldn't work.

Reply:
The moment a migration has been applied to a database, it would be an error to change it. This migration already ran on staging so we can't change it anymore.